chimeraTKApp/src/ChimeraTK/EPICS/FixedScalarRecordDeviceSupport.h

```diff
@@ -137,11 +137,14 @@ class FixedScalarRecordDeviceSupportTrait : public RecordDeviceSupportBase {
   /**
    * Updates the record's TIME field with the specified time stamp.
    */
-  void updateTimeStamp(TimeStamp const &timeStamp) {
+  void updateTimeStamp(VersionNumber const &versionNumber) {
```

Comment:
Same comments here as mentioned earlier...

Reply:
this has been fixed in the same way.